app/experimenter/experiments/tests/test_generate_dashboards.py

```diff
@@ -0,0 +1,68 @@
+import mock
+
+from django.core.management import call_command
+from django.conf import settings
+from django.test import TestCase
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.tests.factories import ExperimentFactory
+
+from redash_client.client import RedashClient
+
+
+class GenerateDashboardsTest(TestCase):
+
+    def setUp(self):
+        dashboard_patcher = mock.patch((
+            'experimenter.experiments.management.commands.'
+            'generate_dashboards.StatisticalDashboard'))
+        logging_patcher = mock.patch((
+            'experimenter.experiments.management.commands.'
+            'generate_dashboards.logging'))
+        self.mock_logger = logging_patcher.start()
+        self.MockExperimentDashboard = dashboard_patcher.start()
+        self.addCleanup(logging_patcher.stop)
+        self.addCleanup(dashboard_patcher.stop)
+
+        self.experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_LAUNCHED)
+
+    def test_dashboard_object_generated(self):
+        expected_call_args = (
+            settings.AWS_ACCESS_KEY,
+            settings.AWS_SECRET_KEY,
+            settings.S3_BUCKET_ID_STATS,
+            self.experiment.project.name,
+            self.experiment.name,
+            self.experiment.slug)
+        expected_public_url = "www.some_dashboard_url.com"
```

Comment:
nit: single quotes

Reply:
Also usually you should do something like 'www.example.com/some_dashboard_url' so that there's never a chance it's accidentally a real URL